PerfNext/app/apis/machines.js

```diff
@@ -15,7 +16,19 @@ module.exports = function(app) {
      * Retrieve all the HW related info about all machines
      */
     app.get('/api/machinesInfo', function(req, res) {
-        res.sendFile(global.appRoot + '/config/master_machine_list.xml');
+    	var getOptions = {
+				url: global.APP_DATA.machine_list_url,
```

Comment:
Indentation isn't right still. Can you please refer to `PerfNext/app/apis/builds.js` to see how we indent the code?